dunwoody_disaster/views/CharacterSelector.py

```diff
@@ -17,45 +20,112 @@ def __init__(self, characters: list[Character]):
         super().__init__()
         self.selected = None
         self._callback = DD.unimplemented
-        layout = QVBoxLayout()
+        self.setStyleSheet("background-color: #2f2f2f;")
+
+        row = 0
+
+        layout = QGridLayout(self)
```

Comment:
It shouldn't take self. Habit, I make everything self in my applications for ability to control from main easier and to avoid navigating scope unnecessarily, then name things specifically for easy readability. The style you chose works easier here for copy pasting and standardizing. I will correct.